### DISCUSSION TITLE
[project beta] Unable to upload files to a draft issue

### DISCUSSION THREAD

**Discussion by elitan** [community/community#42918] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

I'm getting a 500 back when trying to upload a file to a draft issue.

I need to first create the issue to upload any file.

Saving text works fine in draft mode. It's just files that are causing an issue.

**Comment by jfullstackdev** [comment, marked as answer]
how big is that file

because 

![image](https://user-images.githubusercontent.com/47092464/210061275-46a396b3-9b33-4967-a127-72b2863f22de.png)

**Reply by elitan** [reply]
That seems to be it. 

Not sure if your link is relevant because it seems to be related to git repositories (not attachments for issues).

never the less, I was able to upload smaller files to drafted GitHub issues.
